proto_parser/src/proto_value_test.cc

```diff
@@ -23,7 +23,142 @@
 
 namespace wireless_android_play_analytics{
 
-// TODO(alexanderlin): Implement.
+const std::string text_proto = R"pb(
+# comment 1
+int32_field: 1
+
+# comment 2
+field_nested_message { # comment 3
+  int64_field: 100
+}
+
+field_nested_message {
+  # comment 4
+  # comment 5
+  int64_field: 100
+}
+
+
+bool_field: true
+bool_field: false # comment 6
+)pb";
+
+const std::string well_formatted_text_proto = R"pb(# comment 1
+int32_field: 1 
+# comment 2
+field_nested_message { # comment 3
+  int64_field: 100 
+}
+field_nested_message { 
+  # comment 4
+  # comment 5
+  int64_field: 100 
+}
+bool_field: true 
+bool_field: false # comment 6
+)pb";
+
+class ProtoValueTest : public ::testing::Test {
+ protected:
+  
+  virtual void SetUp() {
+    message = ProtoValue::Create(text_proto, test);
+  }
+
+  // virtual void TearDown() {}
+
+  TestProto test;
+  std::unique_ptr<ProtoValue> message;
+};
+
+TEST_F(ProtoValueTest, CreateTest) {
```

Comment:
Let's talk about how to write test in meeting.

Reply:
Done